### DISCUSSION TITLE
Welcome to StreamController Discussions!

### DISCUSSION THREAD

**Discussion by Core447** [StreamController/StreamController#33] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

**Comment by BertConstanty** [comment]
Hi everybody, i am curently using streamcontroler on fedora with a streamdeck XL and it is working great , i would like to buy a streamdeck plus, i have 3 questions, can I use streamcontroler for 2 stream decks? is  the touch screen (on stremdeck plus)  working with streamcontroler? and the turning button (on stremdeck plus) as well? so if someone have the answer it would be great, thank you...

**Reply by oparada1988** [reply]
> Hi everybody, i am curently using streamcontroler on fedora with a streamdeck XL and it is working great , i would like to buy a streamdeck plus, i have 3 questions, can I use streamcontroler for 2 stream decks? is the touch screen (on stremdeck plus) working with streamcontroler? and the turning button (on stremdeck plus) as well? so if someone have the answer it would be great, thank you...


Hi,  I can donfirm that StreamController works with:

1, Multiple device
2. Worls woith Streamdeck plus
3. 3. Touch screen workd


The wolrd is your oyster man!
